sgtm/artemis/README.md

```diff
@@ -0,0 +1,273 @@
+- Realtime on-site personalisation enabled by passing data back to the browser
+using sGTM's advanced tag behaviour
+
+For examples of how to set-up sGTM for these use cases see [XXX BELOW](kahaksjhdkasjhdaksdhaksjhdkasjdhkasjd).
+
+> Tip 💡: Check out [the sGTM Pantheon](
+    https://github.com/google-marketing-solutions/gps-sgtm-pantheon/tree/main)
+for more tags for different use-cases.
+
+## Background
+First party data is becoming ever more important to digital marketing. Marketers
+who are able to effectively use the insights and knowledge they have about their
+customers have an advantage over those who do not use this data. Accessing data
+about customers that is not immediatly available on the website is usually
+difficult though and requires significant web development and involves exposing 
+data on the website. This means many digital marketing teams do not make use of 
+the extensive data their BI teams have on their customers
+
+This data is sensitive, therefore most clients would not like a determined user 
+to be able to access this information. As a result, clients will tend to either 
+use a proxy value in its place or implement a batch upload strategy using one of 
+the Google APIs. The former complicates reporting and some clients are uncomfortable 
+with this approach. The latter poses issues due to the lack of support for 
+conversion modelling, and some worry about the impact on bidding, caused by the 
+delay in reporting the conversions in batches.
+
+sGTM has feature parity for modelling with the client side JavaScript tag, it
+happens in real time, and enables pulling in external "sensitive" data, so it
+operates in the sweet spot between the two existing approaches.
+
+## Why Artemis?
+
+[Artemis](https://en.wikipedia.org/wiki/Artemis) is the Greek goddess of the hunt. 
+Like the goddess, the solution goes out hunting for data and returns with it.
+
+## Implementation
+Ready to start implementing this solution? You can follow the guide below which 
+outlines how Server Side Google Tag Manager (sGTM) can be used with Firestore, 
+to pull in sensitive data and report access it in sGTM variables which can then
+be used for the uses cases listed above.
+
+## Prerequisites
+
+-   Server Side Google Tag Manager
+-   Access to a Google Cloud project with Firestore in
+    [Native mode](https://cloud.google.com/datastore/docs/firestore-or-datastore)
+-   Access to user data keyed by a user ID available within the dataLayer of
+your website
+
+## Architecture
+
+[Image 1](#image-1) provides a high level overview of the core components
+involved, and outlines the flow. This is high level, please see
+[solution details](#solution-details) for further information.
+
+![image1.png](./img/artemis_high_level_architecture.png)
+
+##### *Image 1*
+
+<br>
+
+1.  A GTM web container is used to set up tagging on the site, and is configured
+    with a dataLayer event ([more details on gTag events]
```

Comment:
This link isn't rendering correctly. I think the line break after the `[` is causing the issue